applications/SolidMechanicsApplication/custom_python/solid_mechanics_python_application.cpp

```diff
@@ -136,6 +137,11 @@ BOOST_PYTHON_MODULE(KratosSolidMechanicsApplication)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE( SHEARxREDUCED_AREA )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE( SHEARxPOLAR_INERTIA )
 
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( NEWMARK_BETA )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( NEWMARK_GAMMA )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( BOSSAK_ALPHA )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( EQUILIBRIUM_POINT )
```

Comment:
Are these variables used in any script?

Reply:
Yes, right now, only some of them. This is why I need to register them in python. It is a little bit confusing because they are core variables, but not registered in python by the core. So from python scripts I have to call them  belonging to the  app that has registered them in python.

Reply:
if these are variables of the core, than wouldn't it be better to register them to python in the core itself?

Reply:
If it is ok I will register them in the core